azure/app/parameters/test.template.json

```diff
@@ -144,6 +144,28 @@
     },
     "ROLLBAR_ENV": {
       "value": "test"
+    },
+    "GOVUK_VERIFY_ENABLED": {
+      "value": ""
+    },
+    "VSP.VERIFY_ENVIRONMENT": {
+      "value": "INTEGRATION"
+    },
+    "VSP.SAML_SIGNING_KEY": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "VspSamlSigningKey"
+      }
+    },
+    "VSP.SAML_PRIMARY_ENCRYPTION_KEY": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "VspSamlPrimaryEncryptionKey"
+      }
```

Comment:
Let's make sure we record that we need to change these keys over when the new certs are approved (a Trello card?). Does updating the versions also go in the docs somewhere?

Reply:
I've created a new Trello card for it: https://trello.com/c/YD86WbIp

I'll check about the normal process for rotating certs